### DISCUSSION TITLE
Bass enh. freq.

### DISCUSSION THREAD

**Discussion by glido1** [ElectroTechnique/electrotechnique.github.io#51] [Hardware Issues]
Question.
What frequency is the Bass enhancer peak fixed at?
Too high in my opinion. I would boost from 70-80Hz / +3-6dB to 20Hz.
I'm listening to the APS AEON 2 and SW Yamaha YST 160SW. The APS themselves already show that Bass enh unnecessarily hooks the bass in its upper range, only starting the 160SW gives clear evidence for the thesis discussed. Bass enh. it is useful in many sonic parts.

**Comment by Electrotechnique** [comment]
You can change it!
` sgtl5000_1.enhanceBass(0.85, 0.87, 0, 4);//Normal level, bass level, HPF bypass (1 - on), bass cutoff freq`
See [https://www.pjrc.com/teensy/gui/index.html?info=AudioControlSGTL5000](https://www.pjrc.com/teensy/gui/index.html?info=AudioControlSGTL5000)
